app/controllers/campaigns_controller.rb

```diff
@@ -64,6 +64,15 @@ def users
     ).includes(:user, :course).order(revision_count: :desc)
   end
 
+  def alerts
```

Comment:
@ragesoss I would like to have your opinion here. I created a new method in the campaigns controller that returns an array with alerts method. This way we'll be able to call sth like "this.props.campaign.alerts" in the alerts_handler (react). What do you think of this approach?

Thanks in advance!

Reply:
This makes sense, but you want to be able to do something like @campaign.alerts, which should come strictly from scopes.